Solution169.cs

```diff
@@ -0,0 +1,30 @@
+public class Solution169 {
+    public int MajorityElement(int[] nums) {
+        
+        var dict = new Dictionary<int, int> ();
+
+        foreach (var num in nums)
+        {
+            if (!dict.ContainsKey(num))
+            {
+                dict[num] = 1;
+            }
+            else
+            {
+                dict[num]++;
+            }
+        }
+
+        int length = nums.Length/2;
```

Comment:
I would suggest to use spaces before and after math operators.

Reply:
ok